test/test_translate.py

```diff
@@ -93,3 +93,82 @@ def testORF1abPolyprotein(self):
         # K   'repeats' times, then  N   L   N   P   L   K
         expected = 'K' * repeats + 'NLNPLK'
         self.assertEqual(expected, translate(seq, 'ORF1ab polyprotein'))
+
+
+class TestTranslateSpike(TestCase):
+    """
+    Tests for the translate.translateSpike function.
+    """
+    def testNoGapsCorrectLength(self):
+        """
+        A sequence with no gaps must have the correct length.
+        """
+        seq = 'TTGGTTGTTTATTACCAC'
```

Comment:
What happens when you pass a sequence that is not a multiple of three long?

Reply:
Added test.